.gitignore

```diff
@@ -3,3 +3,4 @@ yarn-error.log
 /dist
 /node_modules
 /temp
+package-lock.json
```

Comment:
using yarn, this shouldn't exist seeing it in git history helps knowing it should be deleted
```suggestion
```